Libraries/LibJS/Runtime/DateConstructor.cpp

```diff
@@ -46,6 +46,12 @@ DateConstructor::~DateConstructor()
 }
 
 Value DateConstructor::call(Interpreter& interpreter)
+{
+    auto* date = static_cast<Date*>(construct(interpreter).as_object());
+    return js_string(interpreter.heap(), date->string());
```

Comment:
The constructor may throw an exception (maybe `OutOfMemoryError`) so I think we need to null-check `date` here and return `{}` in case of failure.